### DISCUSSION TITLE
FriendsOfSymfony 1.5.15: Login in Edge and Chrome not possible

### DISCUSSION THREAD

**Discussion by DirkHorm** [symfony/symfony#49552] [Q&A] [answered]
Hello,

we upgraded our application to PHP 8.2.3 and therefore use the FriendsOfSymfony 1.5.15 fork. The app itself works on Firefox completely, but the login in Edge and Chrome doesn't. A CSRF attack error message pops up trying to login.

In friendsofsymfony1/symfony1/lib/form/sfForm.class.php on line 938

`    return md5($secret.session_id().get_class($this));`

the value returned by session_id() method is different for Edge and Chrome at the point, the login form is created and the token is written to a hidden form field and seconds later when I click the "Login" button.

It seems, for any reason, in Edge and Chrome the session seems to be recreated for every request.

Another issue I saw is, when I call the dev controller on Firefox, I see multiple warnings:

` Warning: session_name(): Session name cannot be changed when a session is active in /*******/lib/vendor/friendsofsymfony1/symfony1/lib/storage/sfSessionStorage.class.php on line 75

Warning: session_set_cookie_params(): Session cookie parameters cannot be changed when a session is active in /*******/lib/vendor/friendsofsymfony1/symfony1/lib/storage/sfSessionStorage.class.php on line 87`

When I call the dev controller in Edge or Chrome, I don't see them, because there doesn't seem to be a session (cookie), yet, that is active.

Hope you can help me. In worst case I would deactivate the CSRF token, but that would only be the last solution.

Thanks and kind regards,

Dirk

**Comment by DirkHorm** [comment]
Comparing the old PHP 7.x php.ini against the PHP 8.2.3 php.ini I saw, that one new session parameter exists

`session.cookie_samesite =""`

It was set to _None_. Changing it to _Lax_ or _Strict_ solved the issue.

**Comment by stof** [comment, marked as answer]
This repo is not about the unofficial `friendsofsymfony1/symfony1` fork of symfony 1 (this repo was never about symfony 1 btw). You should contact the friendsofsymfony1 community for that.

**Reply by DirkHorm** [reply]
Thanks. As written, I already found a solution. My hope was, that people here are also using this fork. Please feel free to delete this thread.
